app/controllers/admin/custom_emojis_controller.rb

```diff
@@ -56,7 +56,7 @@ def copy
       emoji.image = @custom_emoji.image
 
       if emoji.save
-        log_action :create, emoji
+        log_action :copy, emoji
```

Comment:
We [do not have translations for "copy"](https://github.com/tootsuite/mastodon/blob/master/config/locales/en.yml#L148-L176). Copying a custom emoji is essentially creating one. Please revert this